book/src/SDK/overview.md

```diff
@@ -0,0 +1,34 @@
+## SDK
+
+The Harbor SDK is currently comprised of a docker-compose based development environment, the
+`platform` crate, an OpenAPI specification, and code generators. Details of each can be found in 
+their respective subsections, but a brief overview of each is as follows.
+
+### Development Environment
+
+A fully functional development environment can be found in the `devenv` directory. To run the 
+environment, run `docker compose up` from that directory.
+
+### Platform crate
+
+The `platform` crate contains reusable functionality related to completely generic programming 
```

Comment:
I'm afraid to ask, buit can we move platform out yet??  :rofl:

Reply:
Not yet.